src/tail.js

```diff
@@ -1,23 +1,28 @@
 var _checkForMethod = require('./internal/_checkForMethod');
-var _slice = require('./internal/_slice');
+var slice = require('./slice');
 
 
 /**
- * Returns all but the first element of a list. If the list provided has the `tail` method,
- * it will instead return `list.tail()`.
+ * Returns all but the first element of the given list or string (or object
+ * with a `tail` method).
```

Comment:
I parenthesized the last subexpression to avoid ambiguity.

_Returns all but the first element of the given list, string, or object with a `tail` method._

In the sentence above it's unclear whether _with a `tail` method_ relates to _list, string, or object_ or just _object_.